src/components/Upgrade/content.js

```diff
@@ -22,95 +26,139 @@ export const highlights = [
     visual: {
       small: small1,
       large: large1,
-      color: '#ebf5f4',
+      color: 'linear-gradient(135.76deg, #65AAFF 8.69%, #5D21D4 103.74%)',
     },
   },
   {
     title: {
       small: null,
-      large: 'Get activity notifications',
+      large: 'Enhanced voting transparency',
     },
     description: {
       small: null,
       large: `
-        The activity panel keeps you informed about the status of
-        pending transactions and activity in your organization.
+      The voting app has been redesigned to make
+      governance more accessible. Easily find and
+      understand the details of a vote.
       `,
     },
     visual: {
       small: small2,
       large: large2,
-      color: '#d9dfee',
+      color: 'linear-gradient(141.36deg, #A8ED2F -9.7%, #68DFB1 93.28%)',
     },
   },
   {
     title: {
       small: null,
-      large: 'Export your finances in one click',
+      large: 'Stay up to date with email notifications',
     },
     description: {
       small: null,
       large: `
-        Export your finances using a standard data format. Just choose the time
-        period, and click download.
+      Whether it's a new vote, finance transfer, or
+      permissions change, subscribe to the events
+      that matter to you.
       `,
     },
     visual: {
       small: small3,
       large: large3,
-      color: '#fdefe0',
+      color: 'linear-gradient(324.69deg, #FFF886 -112.1%, #FF4E78 91.91%)',
     },
   },
   {
     title: {
       small: null,
-      large: 'Update apps from the App Center',
+      large: 'Beter financial visibility',
     },
     description: {
       small: null,
       large: `
-        Apply security patches and add new features to the apps in your
-        organization by updating to the latest version of apps in the App Center.
+      View and sort your organization’s finances for
+      greater clarity and insight. Now with more
+      powerful filters.
       `,
     },
     visual: {
       small: small4,
       large: large4,
-      color: '#dff3ec',
+      color: 'linear-gradient(135.76deg, #65AAFF 8.69%, #5D21D4 103.74%)',
     },
   },
   {
     title: {
       small: null,
-      large: 'Complete responsive view',
+      large: 'Modern governance',
     },
     description: {
       small: null,
       large: `
-        Manage your organization on the go from any web3 browser with the
-        mobile-friendly Aragon web app.
+      Simplified permissions management that
+      makes it easy to understand at a glance who
+      has the authority in your organization.
       `,
     },
     visual: {
       small: small5,
       large: large5,
-      color: '#dcddf0',
+      color: 'linear-gradient(131.84deg, #FF9372 -58.49%, #FFD770 100%)',
+    },
+  },
+  {
+    title: {
+      small: null,
+      large: 'One app, endless possibilities',
+    },
+    description: {
+      small: null,
+      large: `
+      Aragon Agent enables organizations to interact
```

Comment:
This is something users will have to do more manual work to install into their organizations; I'm wondering if we can leave a link to https://hack.aragon.org/docs/guides-use-agent here?

Reply:
I understand your concern but not sure this is the best place - although this may be the only place we got, well no, the app center room for this information.